modules/bound-sa-tokens-configuring.adoc

```diff
@@ -19,9 +19,9 @@ You can configure pods to request bound service account tokens by using volume p
 +
 This step is typically not required if the bound tokens are used only within the cluster.
 +
-[WARNING]
+[IMPORTANT]
 ====
-If you update the `serviceAccountIssuer` field and there are bound tokens already in use, all bound tokens with the previous issuer value will be invalidated. Unless the holder of a bound token has explicit support for a change in issuer, the holder will not request a new bound token until pods have been restarted.
+If you change the service account issuer, the previous service account issuer is still trusted for the next 24 hours.
```

Comment:
```suggestion
If you change the default service account issuer, the previous service account issuer is still trusted for the next 24 hours.
```
Or more verbosely:
```suggestion
If you change the default service account issuer to a custom one, the previous service account issuer is still trusted for the next 24 hours.
```
Adding this comment, because "change" includes change from a previously changed custom issuer back to the default empty "" issuer. In this case, your statement will not be true, previously changed custom issuer will be removed immediately instead of still trusted for 24h.